src/net_processing.cpp

```diff
@@ -944,8 +944,14 @@ class PeerManagerImpl final : public PeerManager
         EXCLUSIVE_LOCKS_REQUIRED(!m_most_recent_block_mutex, peer.m_getdata_requests_mutex, NetEventsInterface::g_msgproc_mutex)
         LOCKS_EXCLUDED(::cs_main);
 
-    /** Process a new block. Perform any post-processing housekeeping */
-    void ProcessBlock(CNode& node, const std::shared_ptr<const CBlock>& block, bool force_processing, bool min_pow_checked);
+    /** Process a new block. Perform any post-processing housekeeping
+     * @param[in]   node                The peer that we received the block from
+     * @param[in]   block               The block
+     * @param[in]   force_processing    Bypass some of our anti-DoS protections
+     * @param[in]   min_pow_checked     The header is known to be part of a sufficiently high-work chain (anti-dos check)
+     * @return                          True if this is a new block
+     */
+    bool ProcessBlock(CNode& node, const std::shared_ptr<const CBlock>& block, bool force_processing, bool min_pow_checked);
```

Comment:
Would be nice to wrap the long lines in the commit message of 1a5b9d4a09 `validation: log which peer sent us a header` to something like 70-80 columns.